### PR TITLE
32 show animal marker on map

### LINKED ISSUE #32: Show animal marker on map
Show animals location on map using package

### PR DESCRIPTION
**Pull request by RensVanGiersbergen**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,3 +1,3 @@
-PROD_BASE_URL=""
+PROD_BASE_URL="http://localhost"
```
Comment (j-vseg) [inline .env.example:1]: Shouldn't the URL's not be something like this: [https://wildlifenl-uu-michi011.apps.cl01.cp.its.uu.nl](https://wildlifenl-uu-michi011.apps.cl01.cp.its.uu.nl). Even though, this is just an example.

File: lib/views/map/map.dart [hunk 2]
```diff
@@ -1,44 +1,235 @@
...
+  Future<Position> _determinePosition() async {
+    bool serviceEnabled;
+    LocationPermission permission;
+
+    serviceEnabled = await Geolocator.isLocationServiceEnabled();
+    if (!serviceEnabled) {
+      return Future.error('Location services are disabled.');
+    }
+
+    // Check location permissions
```
Comment (j-vseg) [inline lib/views/map/map.dart:86]: I make LocationManager class to handle fetching the permission. Maybe refactor your code before to use that class before merging this into development.

**Review by j-vseg — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (MarkvdKort) [reply .env.example:1]: The entire idea of a .env file is that you don't put these kind of variables on github so I would not advise doing that

**Review by MarkvdKort — APPROVED** [review]

File: lib/views/map/map.dart (hunk 2 above)
Comment (RensVanGiersbergen) [reply lib/views/map/map.dart:86]: This LocationManager wasn't available on my branch at the time of creation, I will do so once your feature branch is merged and conflits are resolved.

**Review by j-vseg — APPROVED** [review]

**Comment by RensVanGiersbergen** [comment]
TODO: Use LocationManager

**Pull request merged**